packages/docs/src/pages/qwikcity/routing/slugs.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Qwikcity - Routing
+---
+
+# Qwikcity - Slugs 
+
+Slugs are parts of the URLs that are extracted into parameters. 
+
+Imagine that you have a page with the following URLs where `[skuId]` can be any of the thousands of products that you have in your database. It would be impractical to create a route for each product. Instead, we need to define a slug (a part of the URL) that will be used to extract the `[skuId]`.
+
+- `http://server/sku/[skuId]` (Example: `http://server/sku/1234` or `http://server/sku/xyz-567`)
+- `http://server/sku/[skuId]/details` (Example: `http://server/sku/1234/details`)
+
+```
+- src/
+  - routes/
+    - sku/
+      - [skuId]/
+        - index.js       # http://server/sku/1234
+        - details.js     # http://server/sku/1234/details
+```
+
+
+## Retrieving the slug from the URL
+
+Once we have `[skuId]` in the URL, we need a way to retrieve it. This can be done by using `useLocation()` API.
+
+```typescript
+import { useLocation } from '@builder.io/qwik-city';
+
+export default component$(() => {
+  const loc = useLocation();
+
+  return (
+    <Host>
+      <h1>SKU</h1>
+      <p>pathname: {loc.pathname}</p>
+      <p>skuId: {loc.params.skuId}</p>
```

Comment:
A `slug` is read by accessing `params`.

We can think about if params should be renamed to slug or if we rename this section to `parameters`.
I believe `queryParameters` are or will be supported as well.

`parameters` & `queryParameters` sound more convenient to me.

Reply:
@adamdbradley / @manucorporat What are your thoughts? I think other frameworks use `params` already, so my inclination is to leave as is.